### ISSUE TITLE
Get-ControlUpAgent fails on PowerShell Core

### ISSUE THREAD

**Issue by aaronparker** [aaronparker/Evergreen#6]
**Describe the bug**

`Get-ControlUpAgent` fails with:

```
Exception calling ".ctor" with "1" argument(s): "Input string was not in a correct format."                                                                         
At /Users/aaron/Projects/Evergreen/Evergreen/Public/Get-ControlUpAgent.ps1:58 char:5                                                                                
+     $version = [Version]::new(($splitLines | Select-String -Pattern $ ...                                                                                         
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                                                             
+ CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
+ FullyQualifiedErrorId : FormatException
```

**Comment by aaronparker** [comment]
Works on Windows PowerShell but fails on PowerShell Core

**Comment by aaronparker** [comment]
Fixed in #28
